### ISSUE TITLE
Safe mode - feature request

### ISSUE THREAD

**Issue by sfkleach** [hey-red/Markdown#83]
We would like to use this for user-content editing, which means having a safe-mode to protect against html tags. Would it be possible to add a Safe option?

**Comment by hey-red** [comment]
Yes, it's possible, but currently I'm quite busy. Maybe in the next few days I can implement this.
